graph-subscriptions-api/datasource/src/datsource_postgres.rs

```diff
@@ -411,15 +430,21 @@ impl DatasourceWriter for DatasourcePostgres {
                 let key = String::from_utf8_lossy(msg.key().unwrap_or_default()).to_string();
                 let (start, end) = build_timerange_timestamp(timestamp);
 
+                let ticket_name = match serde_json::from_str::<GatewaySubscriptionQueryResultTicketPayload>(&query_result_msg.ticket_payload) {
+                    Result::Ok(payload) => payload.name,
+                    Err(_) => Some(String::from(""))
+                };
```

Comment:
This is not "better", just an alternative.
```suggestion
                let ticket_name = serde_json::from_str::<GatewaySubscriptionQueryResultTicketPayload>(&query_result_msg.ticket_payload).map(|payload| payload.name).unwrap_or_default();
```

Reply:
Definitely cleaner. Still learning where I can apply `.map`. Thanks!